### ISSUE TITLE
Unable to run freenet-microblogging example 

### ISSUE THREAD

**Issue by minikin** [freenet/locutus#448]
I can't find a way how to run `freenet-microblogging` example. 
I'd appreciate it if someone could point me out what I'm doing wrong.

Here are the results of the steps:

<details>

<summary><b>1. Build</b></summary>

```sh
locutus/apps/freenet-microblogging on  main [!] via 🦀 v1.66.0 took 5m24s
❯ clear

locutus/apps/freenet-microblogging on  main [!] via 🦀 v1.66.0
❯ make build
cd /Users/user/Documents/GitHub/locutus
cd crates/locutus-node
cargo build --bin ldt
    Finished dev [unoptimized + debuginfo] target(s) in 0.20s
cd /Users/user/Documents/GitHub/locutus
cd apps/freenet-microblogging
cd ./web
/Users/user/Documents/GitHub/locutus/target/debug/ldt build
mv ./build/locutus/freenet_microblogging_web.wasm ../../../crates/locutus-node/examples/
mv ./build/locutus/contract-state ../../../crates/locutus-node/examples/freenet_microblogging_web
Compiling contract with rust
    Finished release [optimized] target(s) in 0.05s
Contract compiled
Compiling contract with rust
    Finished release [optimized] target(s) in 0.06s
Contract compiled
Bundling webapp contract state

up to date, audited 574 packages in 1s

62 packages are looking for funding
  run `npm fund` for details

1 high severity vulnerability

To address all issues, run:
  npm audit fix

Run `npm audit` for details.
asset bundle.js 1.91 MiB [compared for emit] [minimized] [big] (name: main)
orphan modules 73.5 KiB [orphan] 15 modules
runtime modules 1.36 KiB 6 modules
cacheable modules 822 KiB
  javascript modules 818 KiB
    modules by path ./ 728 KiB 12 modules
    modules by path ../../stdlib/ 89.9 KiB
      ../../stdlib/src/webSocketInterface.ts 11.1 KiB [built] [code generated]
      + 3 modules
  asset modules 4.42 KiB
    data:image/svg+xml,%3csvg xmlns=%27.. 281 bytes [built] [code generated]
    data:image/svg+xml,%3csvg xmlns=%27.. 279 bytes [built] [code generated]
    data:image/svg+xml,%3csvg xmlns=%27.. 161 bytes [built] [code generated]
    data:image/svg+xml,%3csvg xmlns=%27.. 271 bytes [built] [code generated]
    data:image/svg+xml,%3csvg xmlns=%27.. 181 bytes [built] [code generated]
    + 11 modules

WARNING in configuration
The 'mode' option has not been set, webpack will fallback to 'production' for this value.
Set 'mode' option to 'development' or 'production' to enable defaults for each environment.
You can also set it to 'none' to disable any default behavior. Learn more: https://webpack.js.org/configuration/mode/

WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets:
  bundle.js (1.91 MiB)

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  main (1.91 MiB)
      bundle.js


WARNING in webpack performance recommendations:
You can limit the size of your bundles by using import() or require.ensure to lazy load some parts of your application.
For more info visit https://webpack.js.org/guides/code-splitting/

webpack 5.75.0 compiled with 4 warnings in 3821 ms
Compiled input using webpack
Finished bundling webapp contract state
mv: ./build/locutus/freenet_microblogging_web.wasm: No such file or directory
cd /Users/user/Documents/GitHub/locutus
cd apps/freenet-microblogging
cd ./contracts/posts
/Users/user/Documents/GitHub/locutus/target/debug/ldt build
mv ./build/locutus/freenet_microblogging_posts.wasm ../../../../crates/locutus-node/examples/
mv ./build/locutus/contract-state ../../../../crates/locutus-node/examples/freenet_microblogging_posts
Compiling contract with rust
    Finished release [optimized] target(s) in 0.05s
Contract compiled
Bundling contract state
Finished bundling state
mv: ./build/locutus/freenet_microblogging_posts.wasm: No such file or directory

locutus/apps/freenet-microblogging on  main [!] via 🦀 v1.66.0 took 6s
```
</details>

<details>

<summary><b>2. Run</b></summary>

```sh
locutus/apps/freenet-microblogging on  main [!] via 🦀 v1.66.0
❯ make run
cd /Users/user/Documents/GitHub/locutus
cd crates/locutus-node/examples
cargo run --example contract_browsing
    Finished dev [unoptimized + debuginfo] target(s) in 0.20s
     Running `/Users/user/Documents/GitHub/locutus/target/debug/examples/contract_browsing`
2023-01-05T16:10:26.719656Z  INFO contract_browsing: loading web contract 88aFGQjgg3u3BpzLGJnn4LqAk8qvDw1HzWBv3uMBaY1J in local node
2023-01-05T16:10:26.719923Z  INFO contract_browsing: loading posts contract 5oFVvp5p6CW2ovqrQvWgBnqaAgAHV1w9GNVjpYRXoo7u in local node
2023-01-05T16:10:26.742613Z  INFO locutus_core::contract::handler::sqlite: loading contract store from "/var/folders/tw/zqq443m16l12jcbg84n5nc0r0000gn/T/locutus/db/locutus.db"
2023-01-05T16:10:26.775111Z  INFO wasmer_compiler_cranelift::translator::func_translator: translate(263 bytes, u0:4(i64 vmctx, i32, i32, i32) apple_aarch64)
2023-01-05T16:10:26.775111Z  INFO wasmer_compiler_cranelift::translator::func_translator: translate(72 bytes, u0:698(i64 vmctx) apple_aarch64)
2023-01-05T16:10:26.775111Z  INFO wasmer_compiler_cranelift::translator::func_translator: translate(52 bytes, u0:467(i64 vmctx) apple_aarch64)
2023-01-05T16:10:26.775111Z  INFO wasmer_compiler_cranelift::translator::func_translator: translate(324 bytes, u0:235(i64 vmctx, i32, i32) -> i32 apple_aarch64)
2023-01-05T16:10:26.777462Z  INFO wasmer_compiler_cranelift::translator::func_translator: translate(82 bytes, u0:468(i64 vmctx) apple_aarch64)
2023-01-05T16:10:26.777469Z  INFO wasmer_compiler_cranelift::translator::func_translator: translate(54 bytes, u0:699(i64 vmctx) apple_aarch64)
vmctx, i32, i32, i32) apple_aarch64)
2023-01-05T16:10:26.820080Z  INFO wasmer_compiler_cranelift::translator::func_translator: translate(315 bytes, u0:89(i64 vmctx, i32, i32) apple_aarch64)
2023-01-05T16:10:26.820185Z  INFO wasmer_compiler_cranelift::translator::func_translator: translate(366 bytes, u0:88(i64 vmctx, i32, i32) apple_aarch64)
2023-01-05T16:10:26.837562Z ERROR locutus_core::executor: other error: Error while importing "__wbindgen_placeholder__"."__wbindgen_describe": unknown import. Expected Function(FunctionType { params: [I32], results: [] })

OMITTED SOME LOGS

2023-01-05T16:10:26.850935Z  INFO wasmer_compiler_cranelift::translator::func_translator: translate(80 bytes, u0:563(i64 vmctx) apple_aarch64)
2023-01-05T16:10:26.857472Z  INFO wasmer_compiler_cranelift::translator::func_translator: translate(70 bytes, u0:381(i64 vmctx) apple_aarch64)
2023-01-05T16:10:26.857567Z  INFO wasmer_compiler_cranelift::translator::func_translator: translate(68 bytes, u0:382(i64 vmctx) apple_aarch64)
2023-01-05T16:10:26.863390Z  INFO wasmer_compiler_cranelift::translator::func_translator: translate(69 bytes, u0:124(i64 vmctx, i32, i32) -> i32 apple_aarch64)
2023-01-05T16:10:26.863478Z  INFO wasmer_compiler_cranelift::translator::func_translator: translate(35 bytes, u0:125(i64 vmctx, i32, i32) -> i32 apple_aarch64)
2023-01-05T16:10:26.868496Z ERROR locutus_core::executor: other error: Error while importing "__wbindgen_placeholder__"."__wbindgen_describe": unknown import. Expected Function(FunctionType { params: [I32], results: [] })
2023-01-05T16:10:26.869398Z  INFO Server::run{addr=127.0.0.1:50509}: warp::server: listening on http://127.0.0.1:50509
2023-01-05T16:10:42.222175Z  INFO request{method=GET path=/contract/web/88aFGQjgg3u3BpzLGJnn4LqAk8qvDw1HzWBv3uMBaY1J/ version=HTTP/1.1 remote.addr=127.0.0.1:57647}: warp::filters::trace: processing request
2023-01-05T16:10:42.231750Z ERROR locutus_core::executor: missing contract
2023-01-05T16:10:42.231841Z ERROR locutus::local_node: failed to get contract 88aFGQjgg3u3BpzLGJnn4LqAk8qvDw1HzWBv3uMBaY1J, reason: missing contract
2023-01-05T16:10:42.233244Z ERROR request{method=GET path=/contract/web/88aFGQjgg3u3BpzLGJnn4LqAk8qvDw1HzWBv3uMBaY1J/ version=HTTP/1.1 remote.addr=127.0.0.1:57647}: locutus::web_handling: error getting contract `88aFGQjgg3u3BpzLGJnn4LqAk8qvDw1HzWBv3uMBaY1J`: client error: undhandled error: failed to get contract 88aFGQjgg3u3BpzLGJnn4LqAk8qvDw1HzWBv3uMBaY1J, reason: missing contract
2023-01-05T16:10:42.236552Z ERROR request{method=GET path=/contract/web/88aFGQjgg3u3BpzLGJnn4LqAk8qvDw1HzWBv3uMBaY1J/ version=HTTP/1.1 remote.addr=127.0.0.1:57647}: warp::filters::trace: unable to process request (internal error) status=502 error=None
2023-01-05T16:10:42.268466Z  INFO request{method=GET path=/favicon.ico version=HTTP/1.1 remote.addr=127.0.0.1:57647 referer=http://127.0.0.1:50509/contract/web/88aFGQjgg3u3BpzLGJnn4LqAk8qvDw1HzWBv3uMBaY1J/}: warp::filters::trace: processing request
2023-01-05T16:10:42.268896Z ERROR request{method=GET path=/favicon.ico version=HTTP/1.1 remote.addr=127.0.0.1:57647 referer=http://127.0.0.1:50509/contract/web/88aFGQjgg3u3BpzLGJnn4LqAk8qvDw1HzWBv3uMBaY1J/}: warp::filters::trace: unable to process request (internal error) status=500 error=None
2023-01-05T16:10:50.321258Z  INFO request{method=GET path=/contract/web/5oFVvp5p6CW2ovqrQvWgBnqaAgAHV1w9GNVjpYRXoo7u/ version=HTTP/1.1 remote.addr=127.0.0.1:57647}: warp::filters::trace: processing request
2023-01-05T16:10:50.321917Z ERROR locutus_core::executor: missing contract
2023-01-05T16:10:50.321951Z ERROR locutus::local_node: failed to get contract 5oFVvp5p6CW2ovqrQvWgBnqaAgAHV1w9GNVjpYRXoo7u, reason: missing contract
2023-01-05T16:10:50.326338Z ERROR request{method=GET path=/contract/web/5oFVvp5p6CW2ovqrQvWgBnqaAgAHV1w9GNVjpYRXoo7u/ version=HTTP/1.1 remote.addr=127.0.0.1:57647}: locutus::web_handling: error getting contract `5oFVvp5p6CW2ovqrQvWgBnqaAgAHV1w9GNVjpYRXoo7u`: client error: undhandled error: failed to get contract 5oFVvp5p6CW2ovqrQvWgBnqaAgAHV1w9GNVjpYRXoo7u, reason: missing contract
2023-01-05T16:10:50.330578Z ERROR request{method=GET path=/contract/web/5oFVvp5p6CW2ovqrQvWgBnqaAgAHV1w9GNVjpYRXoo7u/ version=HTTP/1.1 remote.addr=127.0.0.1:57647}: warp::filters::trace: unable to process request (internal error) status=502 error=None
2023-01-05T16:10:50.355731Z  INFO request{method=GET path=/favicon.ico version=HTTP/1.1 remote.addr=127.0.0.1:57647 referer=http://127.0.0.1:50509/contract/web/5oFVvp5p6CW2ovqrQvWgBnqaAgAHV1w9GNVjpYRXoo7u/}: warp::filters::trace: processing request
2023-01-05T16:10:50.355962Z ERROR request{method=GET path=/favicon.ico version=HTTP/1.1 remote.addr=127.0.0.1:57647 referer=http://127.0.0.1:50509/contract/web/5oFVvp5p6CW2ovqrQvWgBnqaAgAHV1w9GNVjpYRXoo7u/}: warp::filters::trace: unable to process request (internal error) status=500 error=None
```
</details>

**Comment by netsirius** [comment]
Hi @minikin!
It seems that the "makefile" file is not up to date. Since the last published version, the file generated in "build/locutus" is no longer just the ".wasm" file. Now it is a file without an extension that contains the bytes referring to the contract version + wasm.

To fix this issue, you should delete the extension ".wasm" in the webapp and posts tasks in the makefile. Specifically, in the "mv" commands after running the "ldt build" command.

I see that the "make run" task also fails, it seems that this is because the example is also not up to date.

In any case, I'm going to fix this now. Thank you for reporting the issue!

**Comment by iduartgomez** [comment]
Fixed by #452
